packages/expo-av/ios/EXAV/EXAV.m

```diff
@@ -61,7 +61,11 @@ @interface EXAV ()
 @property (nonatomic, strong) AVAudioRecorder *audioRecorder;
 @property (nonatomic, assign) BOOL audioRecorderIsPreparing;
 @property (nonatomic, assign) BOOL audioRecorderShouldBeginRecording;
+@property (nonatomic, assign) BOOL mediaServicesReset;
```

Comment:
I would rather name it `mediaServicesDidReset` (or even `mediaServicesDidResetWhileRecording`) to indicate it's a kind-of property of the current recording session.
`mediaServicesReset` suggest it's a flag that would command whether to reset or not `mediaServices`. I would add additional explanation comment about this property purpose here.